Lib/test/test_asyncio/test_events.py

```diff
@@ -1125,12 +1125,18 @@ def test_create_server_ssl_match_failed(self):
         # incorrect server_hostname
         f_c = self.loop.create_connection(MyProto, host, port,
                                           ssl=sslcontext_client)
+
+        # Allow for flexible libssl error messages.
+        regex = "("
+        # OpenSSL
+        regex += "IP address mismatch, certificate is not valid for '127.0.0.1'"
+        regex += "|"
+        # AWS-LC
+        regex += "CERTIFICATE_VERIFY_FAILED"
+        regex += ")"
```

Comment:
I think it would be more idiomatic to use a multi-line string here (using https://docs.python.org/3/library/re.html#re.VERBOSE)

Reply:
Ah, I didn't know about the `re.X` flag. Thanks!